app_test.go

```diff
@@ -2060,11 +2057,6 @@ func TestApp_Run_Help(t *testing.T) {
 			hideHelp:      false,
 			wantContains:  "boom - make an explosive entrance",
 		},
-		{
-			helpArguments: []string{"boom", "help"},
```

Comment:
we still need to keep this behaviour. If command is not found action should default to helpAction

Reply:
I try to remove help command if no sub command.
<pre><code>
	if a.Command(helpCommand.Name) == nil && !a.HideHelp {
		<del>if !a.HideHelpCommand { </del>
		if !a.HideHelpCommand && len(a.Commands) > 0 {
			a.appendCommand(helpCommand)
		}
</code>
</pre>